docsource/releasenotes.rst

```diff
@@ -6,6 +6,10 @@ VERSION: 0.12.0 (Released TBD)
 
 BACKWARDS INCOMPATIBILITIES:
 
+* ``list_objects`` now requires that at least one and no more than 10000 workspaces are
+  specified.
+* As such, the ``excludeGlobal`` and ``perm`` parameters no longer have any use and are
```

Comment:
Are they just being silently ignored? I didn't find anything in the PR that seems to generates errors or warnings for the deprecated params.

Reply:
Yes, silently ignored. I don't want to remove them and have code start failing unnecessarily. There's no way to return warnings to the user without backwards incompatible changes to the API, unfortunately. I could log the usage, but I'm not sure what we'd do with those logs